packages/sqlserver/sodasql/dialects/sqlserver_dialect.py

```diff
@@ -62,7 +62,7 @@ def default_connection_properties(self, params: dict):
             'port': '1433',
             'username': 'env_var(SQLSERVER_USERNAME)',
             'password': 'env_var(SQLSERVER_PASSWORD)',
-            'database': params.get('database', 'your_database'),
+            'database': params.get('database', 'sodasqldb'),
```

Comment:
I guess you didn't intend to leave it here?

Reply:
this is just my local db that I used for testing